MediaBrowser.Providers/Plugins/AudioDb/ArtistImageProvider.cs

```diff
@@ -143,6 +143,7 @@ public Task<HttpResponseInfo> GetImageResponse(string url, CancellationToken can
         }
 
         /// <inheritdoc />
-        public bool Supports(BaseItem item) => item is MusicArtist;
+        public bool Supports(BaseItem item)
+            => Plugin.Instance.Configuration.Enable && item is MusicArtist;
```

Comment:
Why?

Reply:
This will disable image fetches when the plugin is manually disabled.

Reply:
Oh, I somehow missed it is a task